site/docs/components/number-input/examples.mdx

```diff
@@ -10,118 +10,135 @@ data:
 
 ## Default
 
-Number input has an increment step value of 1 and a step block value of 10, and you can only use it to accept numeric characters.
+The `NumberInput` is designed with a default increment step of 1, which is adjustable via the `step` prop for precise value changes. For larger adjustments, users can hold the Shift key while using the Arrow keys to apply a `stepMultiplier`, a prop that defaults to 2.
```

Comment:
The wording here is a little odd: "is designed with" makes it sound like we don't recommend changing it